### ISSUE TITLE
Include changelogs in documentation

### ISSUE THREAD

**Issue by ludovicsteinbach** [ansys/pygranta#121]
See https://actions.docs.ansys.com/version/stable/migrations/docs-changelog-setup.html#include-the-release-notes-in-changelog-md

**Comment by Andy-Grigg** [comment]
I assume this is intended to be applied to all PyGranta packages that use towncrier?

**Comment by ludovicsteinbach** [comment]
Yes that would be for all repos that we maintain that are deploying docs and building a changelog. I created this after a meeting where I got the impression that this was the recommended approach over the changelog.md file.
